mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -295,6 +295,13 @@ class LogisticRegression @Since("1.2.0") (
     instr.logParams(regParam, elasticNetParam, standardization, threshold,
       maxIter, tol, fitIntercept)
 
+    val autoAggregationDepth =
+      if ($(aggregationDepth) > 0) $(aggregationDepth)
+      else getAggregationDepthByFormula(
+        instances.context.getConf.getSizeAsBytes("spark.driver.memory", "1g"),
```

Comment:
We probably dont' want to replicate defaults here. Do something else if it's not set, like return 0 or whatever a "don't know" value is.